src/app/teams/teams.component.ts

```diff
@@ -128,7 +128,7 @@ export class TeamsComponent implements OnInit, AfterViewInit {
     const teamType = this.mode === 'enterprise' ? 'sync' : team.teamType;
     this.teamsService.addTeamDialog(this.user._id, this.mode, { ...team, teamType }).subscribe(() => {
       this.getTeams();
-      const msg = team ? 'Team updated successfully' : 'Team created successfully';
+      const msg = this.mode === 'enterprise' ? 'Enterprise created successfully' : 'Team created successfully';
```

Comment:
The previous condition was to check if the action was to update an existing team/enterprise or to add a new team/enterprise.  Please make sure to incorporate that back in so the message ends with **updated successfully** after updating an existing team/enterprise.

Reply:
The latest commit should fix the issue.